openxla-nvgpu/runtime/src/openxla/runtime/nvgpu/status_util.c

```diff
@@ -0,0 +1,24 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "openxla/runtime/nvgpu/status_util.h"
+
+#include <stddef.h>
+
+#include "openxla/runtime/nvgpu/dynamic_symbols.h"
+
+iree_status_t openxla_cudnn_status_to_status(
+    openxla_cudnn_dynamic_symbols_t* syms, cudnnStatus_t status,
+    const char* file, uint32_t line) {
+  if (IREE_LIKELY(status == CUDNN_STATUS_SUCCESS)) {
+    return iree_ok_status();
+  }
+
+  const char* error_string = syms->cudnnGetErrorString(status);
```

Comment:
This might be `null`, no?

Reply:
I think no: `When an invalid status value is passed to the function, the returned string is CUDNN_UNKNOWN_STATUS`